### PR TITLE
Feature/e UI 3184 add remove pba page phillip whitaker

### PR DESCRIPTION
**Pull request by phillip-whitaker**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/EUI-3184

### Change description ###

Added the UI for add/remove PBA numbers

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/organisation/containers/update-pba-numbers/update-pba-numbers.component.html [hunk 1]
```diff
@@ -0,0 +1,32 @@
+<app-hmcts-main-wrapper [backLink]="'/organisation'" [title]="title">
```
Comment (rayliangatos) [inline src/organisation/containers/update-pba-numbers/update-pba-numbers.component.html:1]: title=title?

File: src/organisation/containers/update-pba-numbers/update-pba-numbers.component.html [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+      <button type="submit" class="govuk-button hmcts-button--secondary">
+        Add another PBA number
+      </button>
+    </div>
+    
+    <button type="submit" class="govuk-button">
+      Continue
+    </button>
+  </div>
+</app-hmcts-main-wrapper>
```
Comment (rayliangatos) [inline src/organisation/containers/update-pba-numbers/update-pba-numbers.component.html:32]: a new line is required

File: src/organisation/containers/update-pba-numbers/update-pba-numbers.component.html (hunk 1 above)
Comment (phillip-whitaker) [reply src/organisation/containers/update-pba-numbers/update-pba-numbers.component.html:1]: I decided to put the title in the TS file, it is a variable reference

File: src/organisation/containers/update-pba-numbers/update-pba-numbers.component.html (hunk 2 above)
Comment (phillip-whitaker) [reply src/organisation/containers/update-pba-numbers/update-pba-numbers.component.html]: actioned

File: src/organisation/containers/update-pba-numbers/update-pba-numbers.component.ts [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+})
+export class UpdatePbaNumbersComponent implements OnInit {
+
+  public readonly title = 'Add or remove PBA accounts';
+
+  public organisationPaymentAccount: string[];
+
+  constructor(private orgStore: Store<fromStore.OrganisationState>) { }
+
+  ngOnInit() {
```
Comment (rayliangatos) [inline src/organisation/containers/update-pba-numbers/update-pba-numbers.component.ts:18]: ts lint is warning needs a public modifier

Comment (phillip-whitaker) [reply src/organisation/containers/update-pba-numbers/update-pba-numbers.component.ts:18]: Okay... I will add it, but reluctantly. I really disagree with adding the public accessor keyword to the implemented methods such as ngOnInit(). That goes against standard angular coding practices.

**Review by rayliangatos — APPROVED** [review]

**Comment by phillip-whitaker** [comment]
Closing as these changes have been placed in: https://github.com/hmcts/rpx-xui-manage-organisations/pull/655

**Pull request closed without merging**